### ISSUE TITLE
Accessibility: Focus indicator missing

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#51838]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

There is no visible focus outline on the country and state comboboxes in the change address section.

![T008_KBOutlineMissing](https://github.com/user-attachments/assets/5b156c19-54d2-42ac-8d27-8e9392c8548f)

### Component or Page: 

Page: Cart (Modern)

### Name affected element:

Change Address Comboboxes, such as:
- Country/Region United States (US)
- State Texas

### Relevant code snippets:

`.wc-blocks-components-select .wc-blocks-components-select__select:focus {
    box-shadow: 0 0 0 1px inherit;
    outline: 0;
}`

### Recommended fix:

Remove the outline: 0 from .wc-blocks-components-select .wc-blocks-components-select__select:focus
